src/core/expression/qgsexpression.cpp

```diff
@@ -830,6 +830,10 @@ void QgsExpression::initVariableHelp()
   sVariableHelpTexts.insert( QStringLiteral( "symbol_color" ), QCoreApplication::translate( "symbol_color", "Color of symbol used to render the feature." ) );
   sVariableHelpTexts.insert( QStringLiteral( "symbol_angle" ), QCoreApplication::translate( "symbol_angle", "Angle of symbol used to render the feature (valid for marker symbols only)." ) );
 
+  sVariableHelpTexts.insert( QStringLiteral( "symbol_label" ), QCoreApplication::translate( "variable_help", "Label of the symbol, user defined label will be used, otherwise will default to the label made by QGIS." ) );
```

Comment:
```suggestion
  sVariableHelpTexts.insert( QStringLiteral( "symbol_label" ), QCoreApplication::translate( "symbol_label", "Label of the symbol, user defined label will be used, otherwise will default to the label made by QGIS." ) );
```